go.mod

```diff
@@ -3,15 +3,18 @@ module github.com/peggyjv/gravity-bridge
 go 1.15
 
 require (
-	github.com/cosmos/cosmos-sdk v0.45.3
+	github.com/cosmos/cosmos-sdk v0.46.0
 	github.com/cosmos/go-bip39 v1.0.0
-	github.com/ethereum/go-ethereum v1.10.11
+	github.com/ethereum/go-ethereum v1.10.17
```

Comment:
it has this issue: https://github.com/advisories/GHSA-rqmg-hrg4-fm69 but that would be more relevant to the orchestrator/relayer which isn't depending on go-ethereum?